packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverBody/TeachingPopoverBody.test.tsx

```diff
@@ -7,6 +7,7 @@ describe('TeachingPopoverBody', () => {
   isConformant({
     Component: TeachingPopoverBody,
     displayName: 'TeachingPopoverBody',
+    requiredProps: { media: { src: './test' } },
```

Comment:
<img width="306" alt="image" src="https://github.com/microsoft/fluentui/assets/14183168/b4b8d454-0d21-49e7-a178-203a9a6a46c0">

It seems to be invalid with latest changes.